Decide/src/main/java/DD2480/decide/CMVHandler.java

```diff
@@ -30,9 +32,22 @@ public CMVHandler(Parameter parameters, Points[] dataPoints) {
 	}
 
 	private boolean licZero() {
-		// TODO Auto-generated method stub
-		return false;
-	}
+        Points dp1, dp2;
+        
+        for(int i = 0; i < this.dataPoints.length - 1; i++) {
+            dp1 = this.dataPoints[i];
+            dp2 = this.dataPoints[i+1];
+
+            double dist = Math.sqrt(Math.pow(Math.abs(dp1.x - dp2.x), 2) + Math.pow(Math.abs(dp1.y - dp2.y), 2));
```

Comment:
Regarding issue #25: Add functionality to Points class, Should we have your distance calculation as a method or perhaps in un other class?

Reply:
Now there is a distance metod in Points.